### PR TITLE
Feature/blui

### PR DESCRIPTION
**Pull request by andrupu**
For # PXBLUE-2510

#### Changes proposed in this Pull Request:
- Rebrand to BLUI

### CODE REVIEW

**Comment by ektasawant** [comment]
Hi @andrupu , please remove pxblue ref ?

- [ ] https://github.com/pxblue/react-design-patterns/blob/dev/public/index.html#L5

- [ ] https://github.com/pxblue/react-design-patterns/blob/dev/public/index.html#L8

- [ ] https://github.com/pxblue/react-design-patterns/blob/dev/public/manifest.json#L2

- [ ] Rename image - https://github.com/pxblue/react-design-patterns/blob/dev/public/pxblue.png

File: src/pages/form-validation/sectioned-form/index.tsx [hunk 1]
```diff
@@ -246,7 +246,7 @@ export const SectionedFormValidation = (): JSX.Element => {
 
     return (
         <>
-            <AppBar data-cy={'pxb-toolbar'} position={'sticky'} classes={{ root: classes.appbarRoot }}>
+            <AppBar data-cy={'blui-toolbar'} position={'sticky'} classes={{ root: classes.appbarRoot }}>
```
Comment (emclaug2) [inline src/pages/form-validation/sectioned-form/index.tsx:249]: There's a couple PXB refs lingering in this file.

File: src/App.tsx [hunk 2]
```diff
@@ -148,15 +148,15 @@ export const App: React.FC = () => {
                     titleContent={<Typography variant={'overline'}>More Resources</Typography>}
                     items={[
                         {
-                            title: 'PX Blue Components',
+                            title: 'Brightlayer UI Components',
                             itemID: 'comp lib',
                             onClick: (): void => {
                                 window.open('https://pxblue-components.github.io/react/', '_blank');
```
Comment (emclaug2) [inline src/App.tsx:154]: Remove the pxblue refs here, replace with brightlayer-ui

**Comment by andrupu** [comment]
If I am not mistaken, it is the group consensus to leave the pxblue logo and icon alone for now.  I remember having a logo re-design ideation session in Miro where @ektasawant came up with a good "UI" logo design.  However, those designs have not been finalized yet.

So manifest.json line 6 and index.html line 5 should remain the same.  When we get a finalized new logo, we can replace the current pxblue.png file with the new one (and update those refs).

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**